### ISSUE TITLE
some processes logging to /var/log instead of /usr/local/pf/logs/

### ISSUE THREAD

**Issue by garci66** [inverse-inc/packetfence#3423]
On a packetfence 8.1.0 system, I am seeing some processes going to /var/log/messages instead of the pf-specific log directory (and they are not being split to specific files).

In particular, the following processes are being found in /var/log/messages:

* api-frontend
* redis-server
* haproxy
* pfdns

I made the following changes to `packetfence.conf` under `rsyslog.d`:

```
[root@bds conf]# diff /etc/rsyslog.d/packetfence.conf.rpmorig /etc/rsyslog.d/packetfence.conf
163c163
< if $programname == "pfdns" then
---
> if $programname contains "pfdns" then
230a231,246
>
> # api-frontend.log
> if $programname == "api-frontend" then
>   -/usr/local/pf/logs/api-frontend.log
> & stop
>
> # redis_server.log
> if $programname == "redis-server" then
>   -/usr/local/pf/logs/redis_server.log
> & stop
>
> # haproxy_portal.log
> if $programname == "haproxy" and $msg contains "portal-http" then
>   -/usr/local/pf/logs/haproxy_portal.log
> & stop
>
```

but since the file is dynamically generated, Im not sure how to preserve these changes.

**Comment by garci66** [comment]
Sorry to be a pain, but this means that some logs can grow quite a bit out of control as they are not regularly rotated.

**Comment by jrouzierinverse** [comment]
You can modify conf/rsyslog.conf.tt to make those changes permanent.

**Comment by garci66** [comment]
Thanks James!

Mananaged to find the proper file to edit `lib/pf/constants/syslog.pm` and proposed a fix (and a related logrotate fix that was skipping one log file) in #3433

Best Regards
